Sources/DataPipeline/CustomerIO.swift

```diff
@@ -1,4 +1,4 @@
-import CioInternalCommon
+@_spi(Internal) import CioInternalCommon
```

Comment:
Thank you! It bothers me big time that these types shows up in the auto complete outside the SDK